### DISCUSSION TITLE
Tab with diff settings?

### DISCUSSION THREAD

**Discussion by r3k2** [Jelmerro/Vieb#522] [Q&A] [answered]
Sorry if this is already in the documentation, I just started to use vieb today and I was so far not able to find
example configs only the ones in the example folder and do not covert my use case.
I have a workadventure room with other peers, with nyxt and qutebrowser I can enable passthru so the keybinds are ignored for certain sites so I can use awsd to move around, is possible to do this somehow in Vieb?  
Also I notice no bookmarks.. can I add settings in the config for each site that way I could maybe find a way to do what I need above? 

Thanks

**Comment by Jelmerro** [comment, marked as answer]
Please see https://github.com/Jelmerro/Vieb/issues/100 and https://github.com/Jelmerro/Vieb/issues/57, as well as the `favoritepages` setting for the next best thing at the moment.
